src/cq_cam/command.py

```diff
@@ -169,106 +148,56 @@ def to_vector(self, origin: cq.Vector, relative=False):
 
 
 class Command(ABC):
-    modal = None
-
-    @abstractmethod
-    def to_gcode(self) -> tuple[str, cq.Vector | None]:
-        """Output all the necessary G-Code required to perform the command"""
-        pass
+    pass
 
 
 class MotionCommand(Command, ABC):
-    max_depth: float | None
+    modal: MotionControl
+    start: CommandVector
+    end: CommandVector
     ais_color = "red"
     ais_alt_color = "darkred"
-    previous_command: MotionCommand | None
-    feed: float | None
-    start: cq.Vector
-    end: CommandVector
-    tab: bool  # TODO not the right place to carry tab information imo?
 
     def __init__(
-        self, end: CommandVector, arrow=False, tab=False, feed: float | None = None
+        self,
+        end: CommandVector,
+        start: CommandVector,
+        arrow=False,
     ):
+        if start is not None:
```

Comment:
Can start be `None` (based on type annotation no?)

Reply:
I I don't add the check some of the tests fail. I could add it to the type annotation